scripts/create-tf-backend.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+set -exuo pipefail
+
+# Create an S3 bucket, to use as a Terraform state backend.
+# See https://www.terraform.io/docs/backends/types/s3.html
+
+account_id=$(aws sts get-caller-identity | jq -r .Account)
+bucket_name="${account_id}-dce-tfstate"
+
+# Check if the bucket already exists
+set +e
+aws s3 ls s3://${bucket_name} > /dev/null 2>&1
+listBucketRes=$?
+if [[ listBucketRes -eq 0 ]]; then
+  echo "Using s3://${bucket_name} as Terraform state backend"
+else
+
+  set -e
+
+  # Create the S3 bucket
+  echo "Creating S3 bucket s3://${bucket_name} to use as terraform state backend... "
+  aws s3api create-bucket \
+    --bucket ${bucket_name} \
+    --acl log-delivery-write > /dev/null
+  echo "done."
+
+  # Operations on the bucket seem to fail if immediately after creating
+  # the bucket.
+  sleep 20
```

Comment:
ouch :(

Reply:
`sleep` is the unsung hero of software 😜